kubecost/templates/kubecost-role-template.yaml

```diff
@@ -1,31 +1,3 @@
-{{- if (.Values.reporting).logCollection -}}
-apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
-metadata:
-  namespace: {{ .Release.Namespace }}
-  name: {{ template "kubecost.serviceAccountName" . }}-log-collection
-  labels:
-    {{- include "kubecost.commonLabels" . | nindent 4 }}
-rules:
-- apiGroups: 
-    - ''
-  resources:
-    - "pods/log"
-  verbs:
-    - get
-    - list
-    - watch
-- apiGroups:
-    - ''
-  resources:
-    - configmaps
-  verbs:
-    - get
-    - list
-    - watch
-    - update
```

Comment:
I omitted this rule from the new Role definition I've set up in `aggregator-role.yaml`. I don't see how we would need it for log collection... could anyone help me double-check this?

Reply:
agree